lib/elixir/lib/agent.ex

```diff
@@ -439,6 +439,15 @@ defmodule Agent do
 
   Note that `cast` returns `:ok` immediately, regardless of whether `agent` (or
   the node it should live on) exists.
+
+  ## Examples
+
+      iex> {:ok, pid} = Agent.start_link(fn -> 42 end)
+      iex> Agent.update(pid, fn state -> state + 1 end)
```

Comment:
Since this is the documentation for `cast`, we should use cast here:

```suggestion
      iex> Agent.cast(pid, fn state -> state + 1 end)
```